From: Michael Persico <michael.a.persico@gmail.com>
Date: Fri, 02 Jun 2023 12:38:12 -0400
Subject: [PATCH 1/1] Makefile: remove -ffast-math flag

BinaryBuilder explicitly disallows unsafe options, lest one repents.
As -ffast-math is not needed when building MUSCLE, it can be safely removed from Makefile.
---

diff --git a/src/Makefile b/src/Makefile
index df16673..5fe6b0f 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -24,7 +24,7 @@ ifeq ($(OS),Darwin)
 	CXX := g++-11
 endif
 
-CXXFLAGS := $(CXXFLAGS) -O3 -fopenmp -ffast-math
+CXXFLAGS := $(CXXFLAGS) -O3 -fopenmp
 
 LDFLAGS := $(LDFLAGS) -O3 -fopenmp -pthread -lpthread ${LDFLAGS2}
 
